mailet/api/src/main/java/org/apache/mailet/MailetContext.java

```diff
@@ -266,6 +268,12 @@
      */
     boolean isLocalEmail(MailAddress mailAddress);
 
+    default Collection<MailAddress> localRecipients(Collection<MailAddress> recipients) {
```

Comment:
the whole commit seems remotly related the commit message. I don't understand what is done in this commit and why

Reply:
Read the jira ticket.

I did loads of effort to explain my approach in detail.

Here is a clickable link: https://issues.apache.org/jira/browse/JAMES-3147